src/app/authentication/commons/constants/constants.go

```diff
@@ -8,5 +8,6 @@ const (
 
 // Success messages for password update
```

Comment:
change comment to // authentication success messages

Reply:
ok